src/test/java/com/icthh/xm/ms/dashboard/web/rest/AtomicBulkDashboardResourceTest.java

```diff
@@ -0,0 +1,137 @@
+package com.icthh.xm.ms.dashboard.web.rest;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.icthh.xm.commons.i18n.error.web.ExceptionTranslator;
+import com.icthh.xm.ms.dashboard.DashboardApp;
+import com.icthh.xm.ms.dashboard.config.SecurityBeanOverrideConfiguration;
+import com.icthh.xm.ms.dashboard.domain.Dashboard;
+import com.icthh.xm.ms.dashboard.repository.DashboardRepository;
+import lombok.SneakyThrows;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.data.web.PageableHandlerMethodArgumentResolver;
+import org.springframework.http.converter.json.MappingJackson2HttpMessageConverter;
+import org.springframework.security.test.context.support.WithMockUser;
+import org.springframework.test.context.junit4.SpringRunner;
+import org.springframework.test.web.servlet.MockMvc;
+
+import java.util.stream.Stream;
+
+import static com.icthh.xm.ms.dashboard.util.FileUtils.readAsString;
+import static java.util.Arrays.asList;
+import static junit.framework.TestCase.assertNull;
+import static org.mockito.MockitoAnnotations.initMocks;
+import static org.springframework.http.MediaType.APPLICATION_JSON;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.delete;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.post;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.put;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+import static org.springframework.test.web.servlet.setup.MockMvcBuilders.standaloneSetup;
+
+@RunWith(SpringRunner.class)
+@WithMockUser(authorities = "SUPER-ADMIN")
+@SpringBootTest(classes = {DashboardApp.class, SecurityBeanOverrideConfiguration.class})
+public class AtomicBulkDashboardResourceTest {
```

Comment:
this test does not match includes filter in gradle. 
please rename 
```
    include '**/*UnitTest*'
    include '**/*IntTest*'
```

Reply:
Thanks!